packages/webpack-inline-constant-exports-plugin/index.js

```diff
@@ -9,9 +9,9 @@ function addConstantExport( module, name, value ) {
 	module.constantExports[ name ] = value;
 }
 
-function removeDependency( module, dep ) {
+function removeDependency( moduleGraph, module, dep ) {
 	module.removeDependency( dep );
-	dep.module.reasons = dep.module.reasons.filter( ( r ) => r.dependency !== dep );
+	moduleGraph.removeConnection( dep );
```

Comment:
https://webpack.js.org/blog/2020-10-10-webpack-5-release/#module-and-chunk-graph